ui/App/ProjectScreen/Source/Anchors.svelte

```diff
@@ -0,0 +1,223 @@
+        Anchors are commits from this project anchored on the Ethereum network,
+        via <a class="typo-link" href="https://gnosis-safe.io/">Gnosis Safe</a> or
+        your wallet.
+      </div>
+      <div
+        class="dismiss typo-link"
+        on:click={() => {
+          $isAnchorHintVisible = false;
+        }}>
+        Dismiss
+      </div>
+    </div>
+  {/if}
+
+  {#if latest}
+    <div class="latest-anchor">
+      <div class="header">
+        <div style="display: flex;">
+          <OrgIdentity
+            style="margin-right: 0.5rem;"
+            registration={latest.registration}
+            orgAddress={latest.orgAddress} />
+        </div>
+        <p class="timestamp">
+          {ethereum.formatAnchorTransactionTimestamp(latest.timestamp)}
+        </p>
+      </div>
+      <div class="metadata">
+        <Icon.Commit />
+        <p class="typo-text-bold">Commit hash</p>
+        <IdentifierLink
+          params={{
+            type: "commitHash",
+            hash: latest.commitHash,
+            onClick: () => {
+              openCommit(latest.commitHash, latest.projectId);
+            },
+          }} />
+      </div>
+      <div class="metadata">
+        <Icon.Ethereum />
+        <p class="typo-text-bold">Transaction hash</p>
+        <IdentifierLink
+          params={{
+            type: "transactionHash",
+            hash: latest.transactionId,
+            url: org.etherscanUrl(latest.transactionId),
+          }} />
+      </div>
+    </div>
+  {/if}
+
+  {#if oldAnchors.length > 0}
+    <div class="anchor-list">
+      {#each oldAnchors as anchor}
+        <div class="anchor">
+          <div class="left">
+            <div style="display: flex;">
+              <OrgIdentity
+                style="margin-right: 0.5rem;"
+                registration={anchor.registration}
+                orgAddress={anchor.orgAddress} />
+            </div>
+            <div style="display: flex;">
+              <Icon.Commit />
+              <IdentifierLink
+                params={{
+                  type: "commitHash",
+                  hash: anchor.commitHash,
+                  onClick: () => {
+                    openCommit(anchor.commitHash, anchor.projectId);
+                  },
+                }} />
+            </div>
+            <div style="display: flex;">
+              <Icon.Ethereum />
+              <IdentifierLink
+                params={{
+                  type: "transactionHash",
+                  hash: anchor.transactionId,
+                  url: org.etherscanUrl(anchor.transactionId),
+                }} />
+            </div>
+          </div>
+          <div class="timestamp">
+            {ethereum.formatAnchorTransactionTimestamp(anchor.timestamp)}
+          </div>
+        </div>
+      {/each}
+    </div>
+  {:else}
+    <EmptyState
+      emoji="🚣"
+      text="This project doesn't have any anchors yet. To anchor it, go to one of your orgs and click “Anchor project”." />
```

Comment:
We should only show this if `latest` is `undefined`.

Reply:
de06ecaac0475b76208c99c9455fda9e60121a91